tests/test_nanopub.py

```diff
@@ -1,19 +1,13 @@
 from unittest import mock
-from unittest.mock import patch
 
 import pytest
 import rdflib
-import requests
 from rdflib.namespace import RDF
 
-from nanopub import NanopubClient, namespaces
-from nanopub.nanopub import Nanopub
+from conftest import skip_if_nanopub_server_unavailable
+from nanopub import NanopubClient, namespaces, Nanopub
 
-skip_if_nanopub_server_unavailable = (
-    pytest.mark.skipif(requests.get('http://grlc.nanopubs.lod.labs.vu.nl/').status_code != 200,
-                       reason='Nanopub server is unavailable'))
-
-client = NanopubClient()
+client = NanopubClient(use_test_server=True)
 
 def test_nanopub_construction_with_bnode_introduced_concept():
```

Comment:
@raar1 you mentioned on #21: For testing publishing, yes, although for simple fetching of existing nanopubs at known URIs it might be worth still using the real servers for that, just to indicate to us if something there has changed.

Reply:
I would propose to always use test server when testing. The nanopub test server should actually be the first server to undergo a change, before the real servers change. (but if you think in practice this will not happen we should indeed change back to testing the real servers).

Reply:
Also, if we add support for more search endpoints we can populate the test server with stuff we expect to be returned by searches without polluting the test server.

Reply:
Yes, good point about the test server also being the first place that changes to the API should happen. Could it still be worthwhile keeping a single fetch test that also tries the existing servers? I'm thinking it could maybe help debugging if someone changes existing nanopub servers without it going via our test server first? But otherwise I'm happy enough having everything go via the test server for the reason you mentioned.

Reply:
Yeah, I guess that makes sense. I added one test for querying production server.